src/phase-manager.ts

```diff
@@ -193,6 +196,8 @@ const PHASES = {
 
 export type PhaseConstructorMap = typeof PHASES;
 
+type PhaseInput = [Phase, ...Phase[]];
```

Comment:
```suggestion
type PhaseInput = Readonly<NonEmptyArray<Phase>>;
```
Also requires adding the import.